Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -249,6 +249,24 @@ public void ProcessPullRequest_SomeFilesChanged_ProducesOnlyUnchangedFiles()
             logger.AssertInfoLogged("Incremental PR analysis: 1 files out of 3 are unchanged.");
         }
 
+        [TestMethod]
+        public void ProcessPullRequest_UnexpectedCacheKeys()
+        {
+            using var sut = CreateSut(Mock.Of<IBuildSettings>(x => x.SonarScannerWorkingDirectory == @"C:\ValidBasePath"));
```

Comment:
```suggestion
            using var sut = CreateSut(Mock.Of<IBuildSettings>());
```

Reply:
We need to keep it to to fail properly in case something else goes wrong. Otherwise, `Path.Combine` will fail on 1st argument (base path) and not this key